frontend/crd/src/app/security/user.ts

```diff
@@ -0,0 +1,5 @@
+export interface TempUser {
+    readonly name: string;
+    readonly email: string;
+    readonly role: string;
```

Comment:
We should have an enum for roles (user, faculty, admin)

Reply:
At some point I agree, this is just a temporary class for testing purposes that will be replaced by the actual user object